generated-test/forms/smallProducerDetailsFormProviderSpec.scala

```diff
@@ -0,0 +1,29 @@
+package forms
+
+import forms.behaviours.BooleanFieldBehaviours
+import play.api.data.FormError
+
+class smallProducerDetailsFormProviderSpec extends BooleanFieldBehaviours {
+
+  val requiredKey = "smallProducerDetails.error.required"
+  val invalidKey = "error.boolean"
+
+  val form = new smallProducerDetailsFormProvider()()
+
+  ".value" - {
+
+    val fieldName = "value"
```

Comment:
you should add more tests here like the constraints  we have on the SDIl text box